tests/E2E/test/selectors/BO/catalogpage/Manufacturers/brands.js

```diff
@@ -1,18 +1,18 @@
 module.exports = {
   Brands:{
-    new_brand_button: '//*[@id="page-header-desc-address-new_manufacturer"]',
-    name_input: '//*[@id="name"]',
-    image_input: '//*[@id="logo"]',
-    meta_title_input: '//*[@id="meta_title_1"]',
-    meta_description_input: '//*[@id="meta_description_1"]',
-    meta_keywords_input:'//*[@id="manufacturer_form"]//div[contains(@class,"lang-1")]//div[@class="tagify-container"]/input',
-    active_button: '//*[@id="fieldset_0"]//label[@for="active_on"]',
-    save_button: '//*[@id="manufacturer_form_submit_btn"]',
-    short_description_input: '(//*[@id="manufacturer_form"]//div[@class="mce-tinymce mce-container mce-panel"])[1]',
-    description_input: '(//*[@id="manufacturer_form"]//div[@class="mce-tinymce mce-container mce-panel"])[3]',
-    filter_name_input: '//*[@id="table-manufacturer"]//input[@name="manufacturerFilter_name"]',
-    brand_column: '//*[@id="table-manufacturer"]//tr[%TR]/td[%COL]',
-    brand_search_button: '//*[@id="submitFilterButtonmanufacturer"]',
-    brand_reset_button: '//*[@id="submitFilterButtonmanufacturer"]',
+    new_brand_button: '//*[@id="page-header-desc-configuration-add_manufacturer"]',
+    name_input: '//*[@id="manufacturer_name"]',
+    image_input: '//*[@id="manufacturer_logo"]',
+    meta_title_input: '//*[@id="manufacturer_meta_title_1"]',
+    meta_description_input: '//*[@id="manufacturer_meta_description_1"]',
+    meta_keywords_input:'//*[@id="manufacturer_meta_keyword_1-tokenfield"]',
+    active_button: '//label[@for="manufacturer_is_enabled_1"]',
+    save_button: '//div[@class="card-footer"]/button[contains(text(),"Save")]',
+    short_description_input: '//*[@id="manufacturer_short_description_1"]',
+    description_input: '//*[@id="manufacturer_description_1"]',
+    filter_name_input: '//input[@id="manufacturer_name"]',
+    brand_column: '//*[@id="manufacturer_grid_table"]//tr[%TR]/td[%COL]',
+    brand_search_button: '//button[@name="manufacturer[actions][search]"]',
+    brand_reset_button: '//*[@id="manufacturer_grid_table"]//button[@type="reset"]',
   }
-};
+};
```

Comment:
Please let a blank line at the end of the line, this is [recommended](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file) 😉